src/test/java/com/klarna/hiverunner/sql/split/NewLineUtilTest.java

```diff
@@ -0,0 +1,66 @@
+    assertThat(removeLeadingTrailingNewLines(" \n \n "), is(""));
+    assertThat(removeLeadingTrailingNewLines("\n a \n"), is(" a "));
+    assertThat(removeLeadingTrailingNewLines(" \n a b \n "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(" \n \n a b \n \n "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(" \n a b \n "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(""), is(""));
+    assertThat(removeLeadingTrailingNewLines(" "), is(" "));
+    assertThat(removeLeadingTrailingNewLines(" a "), is(" a "));
+    assertThat(removeLeadingTrailingNewLines("a"), is("a"));
+    assertThat(removeLeadingTrailingNewLines(" a"), is(" a"));
+    assertThat(removeLeadingTrailingNewLines("a "), is("a "));
+    assertThat(removeLeadingTrailingNewLines("\r"), is(""));
+    assertThat(removeLeadingTrailingNewLines(" \r "), is(""));
+    assertThat(removeLeadingTrailingNewLines(" \r \r "), is(""));
+    assertThat(removeLeadingTrailingNewLines("\r a \r"), is(" a "));
+    assertThat(removeLeadingTrailingNewLines(" \r a b \r "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(" \r \r a b \r \r "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(" \r a b \r "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(""), is(""));
+    assertThat(removeLeadingTrailingNewLines(" "), is(" "));
+    assertThat(removeLeadingTrailingNewLines(" a "), is(" a "));
+    assertThat(removeLeadingTrailingNewLines("a"), is("a"));
+    assertThat(removeLeadingTrailingNewLines(" a"), is(" a"));
+    assertThat(removeLeadingTrailingNewLines("a "), is("a "));
+    assertThat(removeLeadingTrailingNewLines("\n\r"), is(""));
+    assertThat(removeLeadingTrailingNewLines(" \n\r "), is(""));
+    assertThat(removeLeadingTrailingNewLines(" \n\r \n\r "), is(""));
+    assertThat(removeLeadingTrailingNewLines("\n\r a \n\r"), is(" a "));
+    assertThat(removeLeadingTrailingNewLines(" \n\r a b \n\r "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(" \n\r \n\r a b \n\r \n\r "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(" \n\r a b \n\r "), is(" a b "));
+    assertThat(removeLeadingTrailingNewLines(""), is(""));
+    assertThat(removeLeadingTrailingNewLines("\t"), is("\t"));
+    assertThat(removeLeadingTrailingNewLines("\ta\t"), is("\ta\t"));
+    assertThat(removeLeadingTrailingNewLines("a"), is("a"));
+    assertThat(removeLeadingTrailingNewLines("\ta"), is("\ta"));
+    assertThat(removeLeadingTrailingNewLines("a\t"), is("a\t"));
+    assertThat(removeLeadingTrailingNewLines("\n"), is(""));
+    assertThat(removeLeadingTrailingNewLines("\t\n\t"), is(""));
+    assertThat(removeLeadingTrailingNewLines("\t\n\t\n\t"), is(""));
+    assertThat(removeLeadingTrailingNewLines("\n\ta\t\n"), is("\ta\t"));
+    assertThat(removeLeadingTrailingNewLines("\t\n\ta\tb\t\n\t"), is("\ta\tb\t"));
+    assertThat(removeLeadingTrailingNewLines("\t\n\t\n\ta\tb\t\n\t\n\t"), is("\ta\tb\t"));
+  }
```

Comment:
Aah, my previous comment about \r test I was actually thinking about this class. Perhaps a \r\f test here just to cover that case too?